pkg/apm/application_instances_test.go

```diff
@@ -0,0 +1,76 @@
+// +build unit
+
+package apm
+
+import (
+	"net/http"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+var (
+	testApplicationInstanceLinks = ApplicationInstanceLinks{
```

Comment:
Not a blocker, but these variables could be moved into the `TestListApplicationInstancesWithParams` function scope to avoid variable name collisions across the test suite.

Reply:
👍